assets/questions/rle.json

```diff
@@ -0,0 +1,135 @@
+{
+  "interview_question": {
+    "title": "Run-Length Encoding",
+    "prompts": [
+      {
+        "instructions": [
+          "Welcome to this programming exercise.",
+          "You will complete this exercise by editing the code in the sample.py file using your favorite editor. Every time you save the file, I will evaluate your code and let you know about next steps.",
```

Comment:
Actually, on re-reading, the second sentence doesn't apply either.

Maybe drop this sentence altogether?